internal/cluster/ready.go

```diff
@@ -34,14 +35,29 @@ func (r *readyState) RefreshStatus(ctx context.Context, c *common.Cluster, db *g
 		return c, nil
 	}
 
-	// Cluster is insufficient
-	mastersInKnown := mappedMastersByRole[intenralhost.HostStatusKnown]
-	if len(mastersInKnown) != minHostsNeededForInstallation {
-		log.Infof("Cluster %s dos not have exactly %d known master hosts, cluster is insufficient.", c.ID, minHostsNeededForInstallation)
-		return updateClusterStatus(log, db, *c.ID, swag.StringValue(c.Status), clusterStatusInsufficient, statusInfoInsufficient)
-
-		//cluster is still ready
-	} else {
-		return c, nil
+	knownMastersHostsCount := 0
```

Comment:
Isn't this duplicated code?

Reply:
Not exactly. but as i wrote in the description it's still work in progress.

Reply:
File was deleted